sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -370,6 +384,26 @@ def _add_singleton_step(self, label, full_label, tag, input_step):
           PropertyNames.OUTPUT_NAME: PropertyNames.OUT}])
     return step
 
+  def run_Impulse(self, transform_node):
+    standard_options = (
+        transform_node.outputs[None].pipeline.options.view_as(StandardOptions))
```

Comment:
Access to pipeline.options is deprecated.  We could use _options to not get the warning. (Access here should be OK, as this not during pipeline construction.)